dashboard/app/controllers/code_reviews_controller.rb

```diff
@@ -59,4 +59,40 @@ def update
 
     render json: code_review.summarize_with_comments
   end
+
+  # GET /code_reviews/from_peers
+  # Returns the list of open code reviews for the given script and level
+  # from peers in the user's code review groups.
+  def from_peers
+    params.require([:scriptId, :levelId])
+
+    code_reviews = CodeReview.open_reviews.where(
+      user_id: peer_user_ids(current_user),
+      level_id: params[:levelId],
+      script_id: params[:scriptId]
+    ).includes(:owner)
+
+    return render json: code_reviews.map(&:summarize_for_peer)
+  end
+
+  # Returns the user ids of the students that are in the same code review group
+  # as the given user.
+  def peer_user_ids(student)
```

Comment:
I think the same user ID can be repeated in what's returned here if two students are in multiple sections together -- I don't think it matters for this use case, but maybe just add a comment to that effect?

Reply:
I'm curious why this helper method is in the controller, is that the right place for it?

Reply:
Good question, I wasn't sure about this either.  I've always thought that our code deals with this inconsistently.  I don't think logic goes in a model because it spans multiple models.  In some controllers, there's enough of this logic that we put it in a _helper file.  In others, I think we just leave it in the controller?

I don't have a strong opinion here (other than that it shouldn't be in the model), where do you think would be a good place to put it?

Reply:
Maureen and I chatted offline and decided to leave this here for now (since we don't have a strong pattern for where to put this logic.)